File: Model/Event.java
package Model;

public class Event {

    private String eventID;
    private String name;
    private String details;
    private String date;
    private String image;
    private String[] members;

    public Event (String eventID, String name, String details, String date, String image) {

        this.eventID = eventID;
        this.name = name;
        this.details = details;
        this.date = date;
        this.image = image;
    }

    public String getEventID() {

        return eventID;
    }

    public String getName() {
        
        return name;
    }

    public String getDetails() {

        return details;
    }

    public String getDate() {

        return date;
    }

    public String getImage() {

        return image;
    }

    public String[] getMembers() {

        return members;
    }
}

File: Model/Event.js
export class Event {
    constructor(eventID, name, details, date, time, image, location) {
        this.eventID = eventID;
        this.name = name;
        this.details = details;
        this.date = date;
        this.time = time;
        this.image = image;
        this.location = location;
        this.members = [];
    }

    getEventID() {
        return this.eventID;
    }

    getName() {
        return this.name;
    }

    getDetails() {
        return this.details;
    }

    getDate() {
        return this.date;
    }

    getTime() {
        return this.time;
    }

    getImage() {
        return this.image;
    }

    getLocation() {
        return this.location;
    }

    getMembers() {
        return this.members;
    }
}